scripts/dependency_config.py

```diff
@@ -0,0 +1,221 @@
+ARCHITECTURES = ['arm', 'arm64', 'amd64', 'ppc64le']
```

Comment:
I'd rather source this from the data (aka take the existing keys in the existing checksums)